packages/terra-slide-group/src/terra-dev-site/test/slide-group/AnimatedSlideGroup.test.jsx

```diff
@@ -36,14 +33,14 @@ class SlideGroupExample extends React.Component {
             {i}
           </h2>
           <br />
-          {i !== 4 ? <button type="button" id={`increment-${i}`} style={{ backgroundColor: 'lightgrey' }} onClick={this.increment}>Increment</button> : null}
-          {i !== 1 ? <button type="button" id={`decrement-${i}`} style={{ backgroundColor: 'lightgrey' }} onClick={this.decrement}>Decrement</button> : null}
+          {i !== 4 ? <button type="button" id={`increment-${i}`} className={cx('button-color')} onClick={this.increment}>Increment</button> : null}
```

Comment:
I'd use `button` or `custom-button` for the class name so that it is re-usable to group other style properties in the future (if need be).

Reply:
fixed here https://github.com/cerner/terra-framework/pull/692/commits/6c706bab37454c6d9c4fb8a8ee2678b5ccc57890.